L1Trigger/L1THGCal/python/hgcalVFEProducer_cfi.py

```diff
@@ -0,0 +1,62 @@
+import FWCore.ParameterSet.Config as cms
+
+from L1Trigger.L1THGCal.hgcalTriggerGeometryESProducer_cfi import *
+
+import SimCalorimetry.HGCalSimProducers.hgcalDigitizer_cfi as digiparam
+import RecoLocalCalo.HGCalRecProducers.HGCalUncalibRecHit_cfi as recoparam
+import RecoLocalCalo.HGCalRecProducers.HGCalRecHit_cfi as recocalibparam 
+import hgcalLayersCalibrationCoefficients_cfi as layercalibparam
+
+# Digitization parameters
+adcSaturation_fC = digiparam.hgceeDigitizer.digiCfg.feCfg.adcSaturation_fC
+adcNbits = digiparam.hgceeDigitizer.digiCfg.feCfg.adcNbits
+tdcSaturation_fC = digiparam.hgceeDigitizer.digiCfg.feCfg.tdcSaturation_fC
+tdcNbits = digiparam.hgceeDigitizer.digiCfg.feCfg.tdcNbits
+tdcOnset_fC = digiparam.hgceeDigitizer.digiCfg.feCfg.tdcOnset_fC
+adcSaturationBH_MIP = digiparam.hgchebackDigitizer.digiCfg.feCfg.adcSaturation_fC
+adcNbitsBH = digiparam.hgchebackDigitizer.digiCfg.feCfg.adcNbits
+
+# Reco calibration parameters
+fCPerMIPee = recoparam.HGCalUncalibRecHit.HGCEEConfig.fCPerMIP
+fCPerMIPfh = recoparam.HGCalUncalibRecHit.HGCHEFConfig.fCPerMIP
+layerWeights = layercalibparam.TrgLayer_dEdX_weights
+thicknessCorrection = recocalibparam.HGCalRecHit.thicknessCorrection
+
+# Equalization in the frontend of the sensor responses to 200um sensors
+frontend_thickness_corrections = [1./(c1*c2) for c1,c2 in zip(fCPerMIPee,thicknessCorrection)]
+c200 = frontend_thickness_corrections[1]
```

Comment:
is this intended to be index 1 rather than index 0?

Reply:
Yes, index `1` is intended, this is the value for 200 microns. `0` corresponds to 100 microns value.